### DISCUSSION TITLE
Extensions / Modules

### DISCUSSION THREAD

**Discussion by xavipolo** [fastapi/fastapi#5363] [Questions] [answered]
Hi all,

Does anyone know if it is planned to have a list (repo, site or anyhing else, official or unoffical) of extensions/modules/integrations for FastAPI?
I recently started using SQLModel, Fastapi-users, but it would be nice to know other modules or integrations that are available for fastapi. Like for example for Web notifications.

I'm pretty sure that at some point in the documentation there was some reference, but I can't find it now.

Thanks

**Comment by iudeen** [comment, marked as answer]
There are a few:

https://github.com/Kludex/awesome-fastapi-projects
https://github.com/florimondmanca/awesome-asgi
https://github.com/fkromer/awesome-fastapi

**Reply by xavipolo** [reply]
thx
